var-test-support/src/main/java/io/varhttp/test/VarClientRequest.java

```diff
@@ -88,4 +98,9 @@ public VarClientRequest contentSerialized(Object content, String contentType) {
 		this.content = writer.toString();
 		return this;
 	}
+
+	public enum ContentFormat {
+		STREAM_CONTENT,
```

Comment:
Couldn't we instead of building logic around this enum always only set input stream on the httpResponse and treat it accordingly in one of the terminating getter (read to string in getContent and return as-is in download method)